packages/framework-integration-tests/integration/cli/cli.project.integration.ts

```diff
@@ -117,52 +128,54 @@ describe('Project', () => {
+    const cartDemoEslintRcContent = readFileContent(CART_DEMO_ESLINT_RC)
     expect(cartDemoEslintRcContent).to.equal(expectedCartDemoEslintRc)
 
-    const expectedCartDemoGitIgnore = await readFileContent('integration/fixtures/cart-demo/.gitignore')
-    const cartDemoGitIgnoreContent = await readFileContent(CART_DEMO_GIT_IGNORE)
+    const expectedCartDemoGitIgnore = readFileContent('integration/fixtures/cart-demo/.gitignore')
+    const cartDemoGitIgnoreContent = readFileContent(CART_DEMO_GIT_IGNORE)
     expect(cartDemoGitIgnoreContent).to.equal(expectedCartDemoGitIgnore)
 
-    const expectedCartDemoPretierRc = await readFileContent('integration/fixtures/cart-demo/.prettierrc.yaml')
-    const cartDemoPretierRcContent = await readFileContent(CART_DEMO_PRETTIER_RC)
+    const expectedCartDemoPretierRc = readFileContent('integration/fixtures/cart-demo/.prettierrc.yaml')
+    const cartDemoPretierRcContent = readFileContent(CART_DEMO_PRETTIER_RC)
     expect(cartDemoPretierRcContent).to.equal(expectedCartDemoPretierRc)
 
-    const expectedCartDemoPackageJson = await readFileContent('integration/fixtures/cart-demo/package.json')
-    const cartDemoPackageJsonContent = await readFileContent(CART_DEMO_PACKAGE_JSON)
+    const expectedCartDemoPackageJson = readFileContent('integration/fixtures/cart-demo/package.json')
+    const cartDemoPackageJsonContent = readFileContent(CART_DEMO_PACKAGE_JSON)
     packageJsonAssertions(
       expectedCartDemoPackageJson.replace(PROJECT_NAME_FIXTURE_PLACEHOLDER, projectName),
       cartDemoPackageJsonContent,
       ['dependencies', 'devDependencies']
     )
+    const cartDemoPackageJsonObject = JSON.parse(cartDemoPackageJsonContent)
+    expect(cartDemoPackageJsonObject['dependencies']['@boostercloud/framework-core']).to.equal(`^${BOOSTER_VERSION}`)
+    expect(cartDemoPackageJsonObject['dependencies']['@boostercloud/framework-types']).to.equal(`^${BOOSTER_VERSION}`)
+    expect(cartDemoPackageJsonObject['devDependencies']['@boostercloud/cli']).to.equal(`^${BOOSTER_VERSION}`)
 
-    const expectedCartDemoTsConfigEslint = await readFileContent('integration/fixtures/cart-demo/tsconfig.eslint.json')
-    const cartDemoTsConfigEslintContent = await readFileContent(CART_DEMO_TS_CONFIG_ESLINT)
+    const expectedCartDemoTsConfigEslint = readFileContent('integration/fixtures/cart-demo/tsconfig.eslint.json')
+    const cartDemoTsConfigEslintContent = readFileContent(CART_DEMO_TS_CONFIG_ESLINT)
     expect(cartDemoTsConfigEslintContent).to.equal(expectedCartDemoTsConfigEslint)
 
-    const expectedCartDemoTsConfig = await readFileContent('integration/fixtures/cart-demo/tsconfig.json')
-    const cartDemoTsConfigContent = await readFileContent(CART_DEMO_TS_CONFIG)
+    const expectedCartDemoTsConfig = readFileContent('integration/fixtures/cart-demo/tsconfig.json')
+    const cartDemoTsConfigContent = readFileContent(CART_DEMO_TS_CONFIG)
```

Comment:
Lots of awaits in non-async methods, I guess it was just garbage from an old refactor.